### PR TITLE
PR to document Explore Metrics (previously datatrails)

### PR DESCRIPTION
**Pull request by Eve832**
This PR will publish the Metrics Explore documentation.

### CODE REVIEW

**Comment by Elfo404** [comment]
@Eve832 @grafana/explore-squad doesn't own data trails (now metrics explore), i think you may want to ask a different team for a review.

**Comment by ifrost** [comment]
This is not an Explore feature 🤔  Probably should be reviewed by a someone working on this project, cc @torkelo / @darrenjaneczek

**Comment by Eve832** [comment]
@piotr1212 I know it's not; reviewers were auto-assigned. I don't really need a review yet from anyone

**Comment by Eve832** [comment]
@Elfo404 I have no control over the auto-assigned reviewers list. you all are CODEOWNERS of these files. I don't need a review from anyone yet

File: docs/sources/explore/metrics-explore.md
Comment (mary-sitzenstatter) [inline docs/sources/explore/metrics-explore.md:39]: I would say: Click "Add Label" to select a label-value pair from the drop-down menu. You can add multiple label-value pairs. You can add multiple label-value pairs. The value will appear above the selected label ...."

**Review by mary-sitzenstatter — COMMENTED** [review]
Hello - I added in a few comments. Please let me know if you have any questions :)

File: docs/sources/explore/metrics-explore.md
Comment (lwandz13) [reply docs/sources/explore/metrics-explore.md:38]: re-worded for clarity

File: docs/sources/explore/metrics-explore.md
Comment (lwandz13) [reply docs/sources/explore/metrics-explore.md:39]: done!

**Comment by lwandz13** [comment]
@Eve832 can you check to make sure I have this backported correctly, the feature is currently in private preview

File: docs/sources/explore/metrics-explore.md
Comment (darrenjaneczek) [inline docs/sources/explore/metrics-explore.md:13]: @mary-sitzenstatter can you confirm that this should be "Grafana Explore Metrics" throughout, vs. "... Metrics Explore"

Comment (mary-sitzenstatter) [reply docs/sources/explore/metrics-explore.md:13]: I would use "Metrics Explore"!

Comment (lwandz13) [reply docs/sources/explore/metrics-explore.md:13]: We need to come to a decision on the name of this feature

Comment (lwandz13) [reply docs/sources/explore/metrics-explore.md]: done - explore metrics and not metrics explore

**Review by mary-sitzenstatter — APPROVED** [review]
LGTM!

**Pull request merged**
